kie-server-parent/kie-server-tests/kie-server-integ-tests-optaplanner/src/test/java/org/kie/server/integrationtests/optaplanner/OptaplannerIntegrationTest.java

```diff
@@ -123,14 +122,14 @@ public void testCreateSolverWrongSolverInstanceConfigPath() throws Exception {
     }
 
     @Test(expected = IllegalArgumentException.class)
-    public void testCreateSolverNullContainer() throws Exception {
+    public void testCreateSolverNullContainer() {
         solverClient.createSolver(null,
                                   SOLVER_1_ID,
-                                  null);
+                                  SOLVER_1_CONFIG);
```

Comment:
This improves mutation coverage because otherwise we don't know if the expected IAE was thrown because of the first null parameter or because of the third parameter.